<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="3">
       <widget class="QPushButton" name="mapErase">
        <property name="text">
         <string>erase</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="mapInsert">
        <property name="text">
         <string>map[key] = value</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="mapPrintAll">
        <property name="text">
         <string>print all</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="mapKey">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>key </string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="mapCount">
        <property name="text">
         <string>count</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QSpinBox" name="mapAtKey">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="mapSize">
        <property name="text">
         <string>size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="mapCountKey">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QSpinBox" name="mapEraseKey">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="mapAt">
        <property name="text">
         <string>operator[]</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="mapClear">
        <property name="text">
         <string>clear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="7" colspan="3">
       <widget class="QTextBrowser" name="mapDisplay"/>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="mapValue">
        <property name="prefix">
         <string>value </string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
